docs/configuring_django.rst

```diff
@@ -41,20 +41,23 @@ Example contents of pytest.ini::
 Order of choosing settings
 --------------------------
 
-When the command option `--ds`, the environment variable and the pytest.ini
-configuration is used at the same time, pytest-django will prefer using
-settings from the command line option `--ds`, then the environment variable and
-last the pytest.ini.
-You can use `addopts = --ds=yourtestsettings` in your pytest configuration
-to automatically add the `--ds` option.
+The order of precedence is, from highest to lowest:
+
+* The command line option ``--ds``
+* The environment variable ``DJANGO_SETTINGS_MODULE``
+* The configuration file - ``pytest.ini``, or file that Pytest finds such as
+  ``tox.ini``
+
+If you want to override from the configuration file, you can use
+``addopts = --ds=yourtestsettings``.
```

Comment:
Mention that `DJANGO_SETTINGS_MODULE` is the name to be used in the config? (when not overriding)

Reply:
Reworded a bit.